content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: How to use Prisma with multiple database schemas
+metaDescription: How to use Prisma with multiple database schemas
+tocDepth: 2
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently a very early, experimental preview feature. Introspection and applying your Prisma schema to your database with `migrate dev` and `db push` are currently not supported. If you rely on them, please do not try this preview feature in your main branch.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently only available with the PostgreSQL and SQL Server connectors.
+
+</Admonition>
+
+This guide explains how to include multiple database schemas in your Prisma schema, and how to query across multiple schemas with Prisma Client.
+
+</TopBlock>
+
+## What are database schemas?
+
+Many database providers allow you to create named collections within a database that group together tables, constraints, indexes and other database features. In PostgreSQL and SQL Server these collections are known as schemas. We will refer to these collections as _database schemas_ to distinguish them from Prisma's own schema file.
```

Comment:
"named collection" is not a standing term for what a schema is. 
As "collection" is actually a thing for MongoDB, I think it is also a not that great choice in general for this context

Reply:
In the 1+ years we have been thinking about this now in the schema team, we never found a good term to describe what a "schema" (or "database" in the MySQL case) really is and how to describe it. We started to call it "bucket" in our docs just to avoid having to use "schema" itself: https://www.notion.so/prismaio/Multi-Schema-Support-e2a20ae4373f4337a48ccf7b3565e282 That of course only works internally unfortunately.

Reply:
Hm, maybe it doesn't make sense to highlight the namespace collision bit, given that it doesn't work in our implementation at the moment?